.github/workflows/test_galaxy_package_integration.yaml

```diff
@@ -0,0 +1,74 @@
+name: Test Galaxy package integration with TPV
+on:
+  push:
+    paths-ignore:
+      - 'client/**'
+      - 'doc/**'
+  pull_request:
+    paths-ignore:
+      - 'client/**'
+      - 'doc/**'
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.10']
+    steps:
+      - name: Checkout galaxy code
+        uses: actions/checkout@v3
+      - name: Checkout tpv code
+        uses: actions/checkout@v3
+        with:
+          repository: galaxyproject/total-perspective-vortex
+          path: 'tpv'
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Get full Python version
+        id: full-python-version
+        shell: bash
+        run: echo "version=$(python -c 'import sys; print("-".join(str(v) for v in sys.version_info))')" >> $GITHUB_OUTPUT
+      # - name: Cache pip dir
+      #   uses: actions/cache@v3
+      #   with:
+      #     path: ~/.cache/pip
+      #     key: pip-cache-${{ matrix.python-version }}-${{ hashFiles('requirements.txt') }}
+      # - name: Cache galaxy venv
+      #   uses: actions/cache@v3
+      #   with:
+      #     path: '.venv'
+      #     key: gxy-venv-${{ runner.os }}-${{ steps.full-python-version.outputs.version }}-${{ hashFiles('requirements.txt') }}-integration
+      - name: Replace tpv galaxy dependencies with package paths from local galaxy build
+        run: |
+          GALAXY_SKIP_NODE=1 GALAXY_SKIP_CLIENT_BUILD=1 make setup-venv
+          # create local release
+          echo "c" | UPSTREAM_REMOTE_URL="https://github.com/nuwang/galaxy.git" ./scripts/release.sh -l
```

Comment:
This is temporarily pointing to my upstream to simulate some additional missing files.